macros/models/_sector/nft/enrich_nft_trades.sql

```diff
@@ -103,7 +103,7 @@ SELECT
         ELSE coalesce(agg_mark.aggregator_name, agg1.name, agg2.name)
         END as aggregator_name
 FROM {{base_trades}} base
-LEFT JOIN {{ref('tokens_nft')}} nft
+LEFT JOIN {{source('tokens', 'nft')}} nft
```

Comment:
fyi @0xRobin 
we are looking to move nft token metadata to tokens subproject, similar to erc20, flipping to source is part of that